packages/plugin-web-attribution/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { WebAttributionPlugin } from './web-attribution';
+export * from './typings/web-attribution';
```

Comment:
same here for type exposure

Reply:
The type will be used in the `analytics-ma-browser`.